qemu/tests/blockdev_commit_top.py

```diff
@@ -0,0 +1,32 @@
+from provider import backup_utils
+
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitTop(BlockDevCommitTest):
+
+    def commit_snapshots(self):
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        options = ["base-node", "top-node", "speed"]
+        arguments = self.params.copy_from_keys(options)
+        arguments["base-node"] = self.get_node_name(device)
+        device = self.get_node_name(snapshot_tags[-1])
+        arguments["top-node"] = device
+        backup_utils.block_commit(self.main_vm, device, **arguments)
```

Comment:
need to check the image chain by 'info block'? I can find the check in manual case

Reply:
have updated case in polarion.